CHANGELOG.md

```diff
@@ -29,15 +29,15 @@
   previously allowed to use spaces in the argument definition without
   parentheses. [This is now a syntax error.][11856]
 - [Native libraries of projects can be added to `polyglot/lib` directory][11874]
-- [Redo stack is no longer lost when interacting with text literals][11908].
```

Comment:
Is this changelog entry removed on purpose?

Reply:
Yes, on purpose. The line is probably a result of bad merge. There already is the [same line](https://github.com/enso-org/enso/blob/6b5818879b3d52019d20ac934714c860296d0467/CHANGELOG.md?plain=1#L10) in the IDE section.